### PR TITLE
adding initial unit tests. fixing inititator to allow creation using new, or directly via function call

### PR DESCRIPTION
**Pull request by mateodelnorte**

### CODE REVIEW

File: test/support/acceptor.properties [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# default settings for sessions
+[DEFAULT]
+ReconnectInterval=60
+PersistMessages=Y
+PostgreSQLStoreDatabase=quickfix
+PostgreSQLStoreUser=electronifie
```
Comment (stefankutko) [inline test/support/acceptor.properties]: Would it make sense to make it something a little more generic here?

File: test/support/initiator.properties [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+[DEFAULT]
+ReconnectInterval=60
+PersistMessages=Y
+FileStorePath=./data
+FileLogPath=./log
+
+[SESSION]
+ConnectionType=initiator
+SenderCompID=NODEQUICKFIX
+TargetCompID=ELECTRONIFIE
```
Comment (stefankutko) [inline test/support/initiator.properties]: Would it make sense to make it something a little more generic?

File: test/index.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    });
+
+    it('should create a new acceptor instance with fix beginString, senderCompID, targetCompID on defined session', function () {
+      var acceptor = new quickfix.acceptor({}, {
+        propertiesFile: acceptorPropertiesFile
+      });
+
+      acceptor.should.not.eql(null);
+      acceptor.getSessions()[0].beginString.should.eql('FIX.4.4');
+      acceptor.getSessions()[0].senderCompID.should.eql('ELECTRONIFIE');
```
Comment (stefankutko) [inline test/index.js]: Would it make sense to make it something a little more generic?

Comment (mateodelnorte) [inline test/index.js]: I've changed these to TargetCompID and SenderCompID in my continuation of this branch. Looks like this just got merged, but it'll change to be more generic with the next PR.

**Pull request merged**